parsec/core/fs/workspacefs/sync_transactions.py

```diff
@@ -43,12 +43,16 @@ def get_filename(manifest: LocalFolderishManifests, entry_id: EntryID) -> Option
 
 
 def get_conflict_filename(
-    filename: EntryName, filenames: List[EntryName], author: DeviceID
+    filename: EntryName, filenames: List[EntryName], content_conflict: bool = False
 ) -> EntryName:
     counter = count(2)
-    new_filename = full_name(filename, f"conflicting with {author}")
+    suffix = "filename conflict"
+    if content_conflict:
+        suffix = "file's content conflict"
```

Comment:
I'd personally use `name conflict` and `content conflict`, until we implement better internationalization for those suffixes.